modules/examples/src/main/java/com/ibm/cloud/dpx_services/dpx/v1/DpxExamples.java

```diff
@@ -0,0 +1,749 @@
+import com.ibm.cloud.dpx_services.dpx.v1.model.GetDraftContractTermsDocumentOptions;
+import com.ibm.cloud.dpx_services.dpx.v1.model.GetInitializeStatusOptions;
+import com.ibm.cloud.dpx_services.dpx.v1.model.GetReleaseContractTermsDocumentOptions;
+import com.ibm.cloud.dpx_services.dpx.v1.model.InitializeOptions;
+import com.ibm.cloud.dpx_services.dpx.v1.model.InitializeResource;
+import com.ibm.cloud.dpx_services.dpx.v1.model.JsonPatchOperation;
+import com.ibm.cloud.dpx_services.dpx.v1.model.ListDataProductDraftsOptions;
+import com.ibm.cloud.dpx_services.dpx.v1.model.ListDataProductReleasesOptions;
+import com.ibm.cloud.dpx_services.dpx.v1.model.ListDataProductsOptions;
+// import com.ibm.cloud.dpx_services.dpx.v1.model.ManageApiKeysOptions;
+import com.ibm.cloud.dpx_services.dpx.v1.model.PublishDataProductDraftOptions;
+import com.ibm.cloud.dpx_services.dpx.v1.model.RetireDataProductReleaseOptions;
+import com.ibm.cloud.dpx_services.dpx.v1.model.UpdateDataProductDraftOptions;
+import com.ibm.cloud.dpx_services.dpx.v1.model.UpdateDataProductReleaseOptions;
+import com.ibm.cloud.dpx_services.dpx.v1.model.UpdateDraftContractTermsDocumentOptions;
+import com.ibm.cloud.sdk.core.http.Response;
+import com.ibm.cloud.sdk.core.service.exception.ServiceResponseException;
+import com.ibm.cloud.sdk.core.util.CredentialUtils;
+import com.ibm.cloud.sdk.core.util.GsonSingleton;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class contains examples of how to use the DPX service.
+ *
+ * The following configuration properties are assumed to be defined:
+ * DPX_URL=&lt;service base url&gt;
+ * DPX_AUTH_TYPE=iam
+ * DPX_APIKEY=&lt;IAM apikey&gt;
+ * DPX_AUTH_URL=&lt;IAM token service base URL - omit this if using the production environment&gt;
+ *
+ * These configuration properties can be exported as environment variables, or stored
+ * in a configuration file and then:
+ * export IBM_CREDENTIALS_FILE=&lt;name of configuration file&gt;
+ */
+public class DpxExamples {
+  private static final Logger logger = LoggerFactory.getLogger(DpxExamples.class);
+  protected DpxExamples() { }
+
+  static {
+    System.setProperty("IBM_CREDENTIALS_FILE", "../../data_product_exchange_api_service_v1.env");
+  }
+
+  /**
+   * The main() function invokes operations of the DPX service.
+   * @param args command-line arguments
+   * @throws Exception an error occurred
+   */
+  @SuppressWarnings("checkstyle:methodlength")
+  public static void main(String[] args) throws Exception {
+    Dpx dpxService = Dpx.newInstance();
+
+    // Load up our test-specific config properties.
+    Map<String, String> testConfigProperties = CredentialUtils.getServiceProperties(Dpx.DEFAULT_SERVICE_NAME);
+
+    // Variables to hold link values
+    String completeADraftByContractTermsIdLink = null;
```

Comment:
Need to address the excessive number of link variables.  This is a symptom of some in-efficient link definitions in the API definition.